docs/components/table.md

```diff
@@ -438,6 +438,7 @@ Pass in an array of header objects for the table.
 | `key`              | string  | A unique key for the column                                                                                                                |
 | `label`            | string  | The label displayed on the table for the column                                                                                            |
 | `sortable`         | boolean | Enables or disables server-side sorting for the column (`false` by default)                                                                |
+| `allowHide`        | boolean | Enable show/hide for this column                                                                                                           |
```

Comment:
nit: can we rename this prop to align with the other properties? 
```suggestion
| `hidable`        | boolean | Enable show/hide for this column                                                                                                           |
```

Reply:
I don't think "hidable" is a word. Besides, this is not a prop but a property within an object that a prop accepts

Reply:
Oh okay, let's make it `hidable` then!